ui/app/AppLayouts/Profile/views/browser/OpenLinksInView.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import StatusQ.Controls
+import StatusQ.Core
+
+ColumnLayout {
+    id: root
+
+    required property var accountSettings
+
+    StatusBaseText {
+        text: qsTr("Open links in")
+    }
+
+    ButtonGroup {
```

Comment:
I believe this buttongroup is not needed, since radio buttons don't use it and mutual exclusivity is actually controlled by the state of `accountSettings.openLinksInStatus`